CMakeLists.txt

```diff
@@ -674,9 +674,9 @@ endif()
 
 if(NOT APPLE AND LLGL_BUILD_RENDERER_VULKAN)
     # Vulkan Renderer
-    include(cmake/FindVulkan.cmake)
-    if(VULKAN_FOUND)
-        include_directories(${VULKAN_INCLUDE_DIR})
+    find_package(Vulkan REQUIRED)
```

Comment:
Using the native `find_package` modules is definitely the preferred way to provide better cross-platform support 👍